src/main/java/ch/njol/skript/expressions/ExprVersionString.java

```diff
@@ -37,43 +32,49 @@
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Version String")
-@Description({"The text to show if the protocol version of the server doesn't match with protocol version of the client. " +
-		"You can check the <a href='#ExprProtocolVersion'>protocol version</a> expression for more information about this.",
-		"This can only be set in a <a href='events.html#server_list_ping'>server list ping</a> event."})
-@Examples({"on server list ping:",
-		"	set the protocol version to 0 # 13w41a (1.7), so it will show the version string always",
-		"	set the version string to \"&lt;light green&gt;Version: &lt;orange&gt;%minecraft version%\""})
+@Description({
+	"The text to show if the protocol version of the server doesn't match with protocol version of the client. " +
+	"You can check the <a href='#ExprProtocolVersion'>protocol version</a> expression for more information about this.",
+	"This can only be set in a <a href='events.html#server_list_ping'>server list ping</a> event."})
+@Examples({
+	"on server list ping:",
+	"	\tset the protocol version to 0 # 13w41a (1.7), so it will show the version string always",
+	"	\tset the version string to \"&lt;light green&gt;Version: &lt;orange&gt;%minecraft version%\""
```

Comment:
```suggestion
	"\tset the protocol version to 0 # 13w41a (1.7), so it will show the version string always",
	"\tset the version string to \"&lt;light green&gt;Version: &lt;orange&gt;%minecraft version%\""
```